package.json

```diff
@@ -9,7 +9,7 @@
     "shrink": "npmshrink",
     "lint": "eslint app server tests --cache",
     "start": "node scripts/check-local-config && grunt server",
-    "start-circle": "NODE_ENV=production grunt build && CONFIG_FILES=server/config/fxaci.json,server/config/production.json grunt serverproc:dist",
+    "start-circle": "CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build && CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt serverproc:dist",
```

Comment:
`start-circle` wasn't working as expected because the correct config wasn't being passed to `grunt build`. I needed to run locally as if on circle to debug the functional tests.